app/(main)/filters.tsx

```diff
@@ -135,12 +135,12 @@ export const tokenFilters: TokenFilter[] = [
     when: {},
   },
   {
-    name: 'wxDai',
+    name: 'WXDAI',
     key: 'wxdai',
     when: { collateralToken_in: [WXDAI.address] },
   },
   {
-    name: 'sDai',
+    name: 'sDAI',
```

Comment:
Maybe we can go full uppercase as CMC and Coingecko do:

https://coinmarketcap.com/es/currencies/savings-dai/